lib/tasks/import.rake

```diff
@@ -159,8 +159,14 @@ def import_without_email(id, first_name, last_name)
     current_user: current_user
   )
 
+  privatable_to = current_process
+
   Decidim::Admin::ImpersonateUser.call(form) do
     on(:ok) do |user|
+      Decidim::ParticipatorySpacePrivateUser.find_or_create_by!(
+        user: user,
+        privatable_to: privatable_to
+      )
       Rails.logger.debug I18n.t("participatory_space_private_users.create.success", scope: "decidim.admin")
       Rails.logger.debug "Registered user with id: #{id}, first_name: #{first_name}, last_name: #{last_name} --> #{user.id}"
```

Comment:
Don't you want to log to a separate file?

Reply:
Not sure ... 
The whole Log / Verbose option workflow should be refactored.